docs/_includes/header-title.adoc

```diff
@@ -5,7 +5,7 @@ Included in:
 ////
 
 A document title must be placed on the first line of the document.
-It is preceded by one equal sign followed by at least one space (`=`), and then the text of the title.
+It is preceded by one equal sign followed by at least one space (`=&nbsp;`), and then the text of the title.
```

Comment:
I'll go ahead and merge and follow-up with that change. But just note that request for future changes.